### DISCUSSION TITLE
Maximum retries exceeded

### DISCUSSION THREAD

**Discussion by frankpablu** [anuraghazra/github-readme-stats#3099] [Q&A] [answered]
### Describe the bug

Please add an env variable callet PAT_ with your github token in vercel

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/85142117/bd93843a-491f-4f92-93b3-4a3879699a8c)

**Comment by rickstaa** [comment, marked as answer]
@frankpablu this is because our public instance is being rate-limited by GitHub (see https://github.com/anuraghazra/github-readme-stats/issues/1471). You can [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) to fix this or wait until the public instance is longer rate-limited.

**Reply by frankpablu** [reply]
@rickstaa Thank you so much
